### ISSUE TITLE
Dev Page Accessibility 

### ISSUE THREAD

**Issue by ADavidson02** [ADavidson02/overlook-project#25]
- [x] Accessibility audits should be at 100%
- [x] A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- [ ] ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
- [x] Application should be fully responsive for mobile - tablet - desktop
